### PR TITLE
Credits what to do view

### PR DESCRIPTION
**Pull request by liampan**
### Description of Work carried through

Brief description of what/why/how.

### CODE REVIEW

File: app/views/returns/credits/WhatDoYouWantToDoView.scala.html [hunk 1]
```diff
@@ -14,30 +14,60 @@
...
+        @headingLabel(messages("what-do-you-want-to-do.title"))
+
+        @paragraphBody(messages("what-do-you-want-to-do.paragraph.1"))
+
+        @govukRadios(
+            RadiosViewModel.apply(
+                field = form("value"),
+                items = Seq(
+                    RadioItem(id = Some(""), value = Some("true"), content = Text(messages("what-do-you-want-to-do.credit-and-return", obligationPrettyPrint))),
+                    RadioItem(id = Some(""), value = Some("false"), content = Text(messages("what-do-you-want-to-do.just-return", obligationPrettyPrint)))
```
Comment (PSouthern) [inline app/views/returns/credits/WhatDoYouWantToDoView.scala.html:55]: What will the id's for these elements be?

Comment (liampan) [reply app/views/returns/credits/WhatDoYouWantToDoView.scala.html:55]: good spot/shout, what would like? 😂 "radio-credit-and-return"  and  "radio-just-return" respectively?

Comment (liampan) [inline app/views/returns/credits/WhatDoYouWantToDoView.scala.html:55]:
```suggestion
                    RadioItem(id = Some("radio-credit-and-return"), value = Some("true"), content = Text(messages("what-do-you-want-to-do.credit-and-return", obligationPrettyPrint))),
                    RadioItem(id = Some("radio-just-return"), value = Some("false"), content = Text(messages("what-do-you-want-to-do.just-return", obligationPrettyPrint)))
```

**Review by C-Thomp — APPROVED** [review]
👍

File: app/controllers/returns/credits/WhatDoYouWantToDoController.scala [hunk 2]
```diff
@@ -40,21 +42,28 @@ class WhatDoYouWantToDoController @Inject() (
   formProvider: WhatDoYouWantToDoFormProvider,
   val controllerComponents: MessagesControllerComponents,
   view: WhatDoYouWantToDoView,
+  taxReturnHelper: TaxReturnHelper,
```
Comment (mauro-fanton) [inline app/controllers/returns/credits/WhatDoYouWantToDoController.scala:45]: Is this used anywhere in this controller

**Review by PSouthern — APPROVED** [review]

**Pull request merged**